src/tools/perf/perftest.c

```diff
@@ -297,6 +300,31 @@ static ucs_status_t setup_sock_rte(struct perftest_context *ctx)
     int sockfd, connfd;
     int ret;
 
+    if (ctx->params.super.flags & UCX_PERF_TEST_FLAG_LOOPBACK) {
+        int connfds[2];
```

Comment:
better split to 2 functions:
1. setup_sock_rte_p2p
2. setup_sock_rte_loobback
then:
```
if (ctx->params.super.flags & UCX_PERF_TEST_FLAG_LOOPBACK) {
  setup_sock_rte_loobkack
} else {
  setup_sock_rte_p2p
}
```